sfepy/discrete/common/extmods/mesh.c

```diff
@@ -927,7 +927,7 @@ inline float64 _tri_area(float64 *coors, uint32 *indices, int32 nc)
     v1[2] = 0.0;
   }
 
-  for (id = 0; id < nc; id++) {
+  for (id = 0; id < (uint32)nc; id++) {
```

Comment:
uint32 nc in the function definition

Reply:
Dtto...

Reply:
Yes, basically, all `D`, `dim`, `nc` can/should be changed to uint32.